lib/perl/Genome/Model/Build/Command/Start.pm

```diff
@@ -143,6 +143,9 @@ sub create_and_start_build {
 
     my $outer_transaction = UR::Context::Transaction->begin();
     $model->build_requested(0);
+    my $anp = $model->analysis_project;
+    my $config_dir = $anp? $anp->environment_config_dir : undef;
```

Comment:
Yeah, I don't think we should hold up this PR for what amounts to an unrelated change. Just something I noticed while reading though.